### PR TITLE
Guinea corrected gadm

### PR DESCRIPTION
**Pull request by espy**
Ok, I went through all the sub-prefectures one by one and compared them to the GADM dataset to ensure that their IDs stay the same. The old dataset still has a couple of extra sub-prefectures, but that shouldn't be a problem. 

I full expect GADM to have errors as well, but since there's no single source of truth, this is as good as it gets for now. 

As a reminder, this was the bug report:

> the subprefectures of Lola are actually those of Macenta, those of Guékédou are actually the ones from Lola, the ones from Macenta are missing.

So this should have happened: 
- sub-prefectures with `parentId` LO (Lola) changed to `parentId` MC (Macenta)
- sub-prefectures with `parentId` GU (Guékédou) changed to `parentId` LO (Lola)
- sub-prefectures with `parentId` MC have magically appeared

Sprintly issue: https://sprint.ly/product/22465/item/1190

### CODE REVIEW

**Comment by danse** [comment]
A change log starts to become useful here. What do you think about starting one? You can use any documented format, like [this](http://keepachangelog.com/) for example. This is a breaking release i think

**Comment by espy** [comment]
Good point. Although I wouldn't call them breaking, since the IDs stay the same, the frontend apps will still display the same level 3 name (unless that itself was wrong), and just assign it to the correct level 2.

But yeah, changelog. Will do later!

File: json/guinea.json [hunk 1]
```diff
@@ -803,12 +803,12 @@
...
         "id": 117,
         "name": "guéassou",
-        "parentId": "YO"
+        "parentId": "LO"
       },
       {
         "id": 118,
         "name": "guéckédou-centre",
-        "parentId": "YO"
+        "parentId": "Gu"
```
Comment (danse) [inline json/guinea.json]: this has the wrong case

**Comment by danse** [comment]
Great work! As we said, it would be better to find a way to print the hierarchy and send it over to Guinea for validation, let's wait for this before merging

**Comment by espy** [comment]
I merged the readable.js into here so it's easier to check the new guinea data.

**Comment by danse** [comment]
great thanks, let's wait for feedback from Nils

**Comment by espy** [comment]
By the way, the readable regions for Guinea are here, in case someone in Guinea wants to review them: https://gist.github.com/espy/19e73aeea7490d2fb2b7

**Comment by skoni** [comment]
@espy Can you bump the version, since it we trigger a reindex as stated here: https://sprint.ly/product/22465/item/1190

**Comment by skoni** [comment]
:+1:

**Pull request merged**